reana_job_controller/variables.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of REANA.
+# Copyright (C) 2017, 2018 CERN.
```

Comment:
Great to see your contribution!  

- Please amend copyright years. 
- Please add yourself and @khurtado to the author list  in `AUTHORS.rst`.
- Regarding merging, we usually merge feature branches with linear history.  There are basically two options: (1) Shall we rebase and keep the history? (2) Shall we squash everything under one commit? What would you prefer?

Reply:
I'll add us to `AUTHORS.rst`! 

Regarding merging, we are fine with whatever would be easier for the REANA team. If squashing everything under one commit is easier for you then we are fine with that, the history of our commits is not as important to us as integrating them.